index.html

```diff
@@ -200,8 +208,11 @@ <h2 class="main-heading">Input</h2>
             <input class="textbox" placeholder="Input Text" />
             <div>Input Doesn't match</div>
           </div>
-          <div class="center-heading-float">Above are the inputs, which can used in building of a web site. Below is the html code which I used to implement them.</div>
-          <img class="html-image" src="code/input-html.png"/>
+          <div class="center-heading-float">
+            Above are the inputs, which can used in building of a web site.
+            Below is the html code which I used to implement them.
```

Comment:
To improve the usability of buttons, add the `cursor: pointer` property on hover.